### PR TITLE
Adding fulfillment sdk

### PR DESCRIPTION
**Pull request by mkytkajrkibo**

### CODE REVIEW

File: fulfillment-sdk/README.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
+# kibo.sdk.java.inventory
+
+Inventory
```
Comment (meganschwind-kibo) [inline fulfillment-sdk/README.md]: Oops... needs to be a fulfillment readme

File: fulfillment-sdk/bin/pom.xml [hunk 2]
```diff
@@ -0,0 +1,206 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.kibocommerce</groupId>
+    <artifactId>inventory-sdk</artifactId>
+    <packaging>jar</packaging>
+    <name>inventory-sdk</name>
```
Comment (meganschwind-kibo) [inline fulfillment-sdk/bin/pom.xml]: fulfillment

File: fulfillment-sdk/bin/settings.gradle [hunk 3]
```diff
@@ -0,0 +1 @@
+rootProject.name = "inventory-sdk"
```
Comment (meganschwind-kibo) [inline fulfillment-sdk/bin/settings.gradle]: fulfillment

File: fulfillment-sdk/bin/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<manifest package="com.kibocommerce.sdk.inventory" xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (meganschwind-kibo) [inline fulfillment-sdk/bin/src/main/AndroidManifest.xml]: fulfillment

**Review by meganschwind-kibo — CHANGES_REQUESTED** [review]
I think you generated with an inventory config file...

File: fulfillment-sdk/README.md (hunk 1 above)
Comment (mkytkajrkibo) [reply fulfillment-sdk/README.md]: This is the culprit

INFO  o.o.codegen.DefaultCodegen - Skipped overwriting README.md as the file already exists in /Users/michael.kytka/IdeaProjects/kibo.sdk.java/fulfillment-sdk/.//README.m

File: fulfillment-sdk/bin/pom.xml (hunk 2 above)
Comment (mkytkajrkibo) [reply fulfillment-sdk/bin/pom.xml]: actually don't know where the bin directory came from. If I delete it and then regenerate it is not regenerated. I think this was left over from a copy and paste of the inventory sdk.

File: fulfillment-sdk/bin/settings.gradle (hunk 3 above)
Comment (mkytkajrkibo) [reply fulfillment-sdk/bin/settings.gradle]: same thing...don't know what bin is being used for and why its not generated by the open api generator

File: fulfillment-sdk/bin/src/main/AndroidManifest.xml (hunk 4 above)
Comment (mkytkajrkibo) [reply fulfillment-sdk/bin/src/main/AndroidManifest.xml]: same...don't need Android anyways

**Comment by mkytkajrkibo** [comment]
I actually did generate with the right configs it would appear. The README.md was skipped over on generation. Should have caught that.

File: fulfillment-sdk/bin/.project [hunk 5]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>inventory-sdk</name>
```
Comment (meganschwind-kibo) [inline fulfillment-sdk/bin/.project]: I don't think you removed the /bin folder?

File: fulfillment-sdk/docs/CreatePickWave.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
+
+# CreatePickWave
```
Comment (meganschwind-kibo) [inline fulfillment-sdk/docs/CreatePickWave.md:2]: invenotry relic. I think you need to do a fresh clean generate in a CLEAN directory

Comment (meganschwind-kibo) [reply fulfillment-sdk/docs/CreatePickWave.md:2]: oh wait we moved pickwaves to fulfillment haha, JK

Comment (mkytkajrkibo) [reply fulfillment-sdk/docs/CreatePickWave.md:2]: lollll

File: inventory-sdk/bin/regen.sh [hunk 7]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
Comment (meganschwind-kibo) [inline inventory-sdk/bin/regen.sh]: why is there a /bin folder under inventory-sdk now?

**Review by meganschwind-kibo — APPROVED** [review]

**Pull request merged**